pkg/tfgen/generate_nodejs.go

```diff
@@ -479,6 +479,29 @@ func (g *nodeJSGenerator) emitResourceType(mod *module, res *resourceType) (stri
 		w.Writefmtln("")
 	}
 
+	w.Writefmtln("    private static readonly __pulumiType = '%s';", res.info.Tok)
```

Comment:
Can we make this public?  That would possibly address https://github.com/pulumi/pulumi/issues/2754.`.

If we do make it pubic - I suspect it is just `type`?

cc also @CyrusNajmabadi for thoughts.

Reply:
I think it would be ok to have this public.

Reply:
@hausdorff FWIW - I don't think this feedback was addressed?